python/py-graveyard/Portfile

```diff
@@ -169,6 +169,7 @@ py-spyder               2.3.7_1     33
 py-spyder-devel         2.3.4_1     33
 py-sqlite               2.6.3_1     26
 py-storm                0.18_1      26
+py-thrift				0.10.0		26 36
```

Comment:
The version needs to be higher, so this needs to be 0.10.0_1 in order to make any difference.
The version 36 should not be here, this is for outdated version only. Even though I have no clue how to change the default to something else than 2.6->2.7.

Plus irrelevant space/tab mismatch.
I'll fix this and commit it.